src/git-town-append

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh"
+
+
+function ensure_has_target_branch {
+  if [ -z "$target_branch_name" ]; then
+    echo_error_header
+    echo_error "No branch name provided."
+    exit_with_error newline
+  fi
+}
+
+
+function preconditions {
+  target_branch_name=$1
+  ensure_has_target_branch
+
+  parent_branch_name=$INITIAL_BRANCH_NAME
+  if [ "$parent_branch_name" != "$MAIN_BRANCH_NAME" ]; then
+    ensure_knows_parent_branches "$parent_branch_name"
+  fi
+
+  if [ "$(has_remote_url)" = true ]; then
+    fetch
+  fi
+
+  ensure_does_not_have_branch "$target_branch_name"
+
+  export RUN_IN_GIT_ROOT=true
+  export STASH_OPEN_CHANGES=true
+}
+
+
+function steps {
+  echo "create_and_checkout_feature_branch $target_branch_name $parent_branch_name"
```

Comment:
By that logic I would say prepend (and even hack) doesn't need it either. Why does a child branch have less need to sync? Just because it was cut off an up to date master previously, doesn't mean it hasn't fallen behind. A lot of git-town is about keeping you up to date with the main branch (between sync and all the syncing the other commands do). I'd lean toward syncing by default, but we could introduce a configuration option for prepend / append to avoid the sync if we want it to be faster.